source/features/parse-backticks.tsx

```diff
@@ -6,7 +6,7 @@ import {parseBackticks} from '../libs/dom-formatters';
 function init(): void {
 	for (const title of select.all(`
 		[aria-label="Issues"][role="group"] .js-navigation-open,
-		.commit-title .js-navigation-open
```

Comment:
Why did you remove this line? This is still needed

Reply:
It should be covered by the `.message` selector

Reply:
Nope. This is for
![image](https://user-images.githubusercontent.com/16872793/71374426-9becf880-2588-11ea-89fc-6e9a3035dc20.png)

Which does not have such a class

Reply:
It doesn't have the `.commit-title` class either. Isn't this covered by `source/features/format-discussion-titles.tsx`?

Reply:
You are correct

Reply:
Given the `js-navigation-open` class id guess that was for isCommitList